cs-cz/sass.html.markdown

```diff
@@ -0,0 +1,439 @@
+---
+language: sass
+filename: learnsass.scss
```

Comment:
Could you also change the filename to "learnsass-cz.scss" or some such thing? Otherwise there can be collisions.